src/popup/views/app.vue

```diff
@@ -141,12 +129,8 @@ export default {
     checkReload() {
       if (options.get('autoReload')) browser.tabs.reload(this.store.currentTab.id);
     },
-    onClickDomains() {
-      if (this.domains.length === 1) {
-        this.onFindScripts(this.domains[0]);
-      } else {
-        this.toggleMenu('domains');
-      }
+    onClickDomain() {
+      this.onFindScripts(this.store.domain);
```

Comment:
`onFindScripts` and `onClickDomain` can be merged into one function.

Reply:
Do you prefer keeping `onFindScripts` or `onClickDomain` function name?

Reply:
I think `onFindScripts` is better